CHANGELOG.unreleased.md

```diff
@@ -19,6 +19,7 @@
 
 - [REST API] Returns a proper 404 _Not Found_ error when a job does not exist instead of _Internal Server Error_
 - [Host/Smart reboot] Automatically retries up to a minute when `HOST_STILL_BOOTING` [#7194](https://github.com/vatesfr/xen-orchestra/issues/7194) (PR [#7231](https://github.com/vatesfr/xen-orchestra/pull/7231))
+- [Mirror backup] Preventing `reportWhen` value from becoming undefined (and resetting to the default value *Failure*) when a mirror backup is edited without modifying this field
```

Comment:
The changelog is user-oriented so this is too technical. It needs to say what visible *behavior* it fixes.